integration-tests/testdata/cpp-multimodule-project/sonar-project.properties

```diff
@@ -0,0 +1,56 @@
+sonar.host.url=http://localhost:9000
+
+sonar.modules=cli,lib,package1,package2
+cli.sonar.projectBaseDir=cli
```

Comment:
Only doxygen is redundant but its not worth it

On Sat, Apr 18, 2015, 08:57 Günter Wirth <notifications@github.com> wrote:

> In
> integration-tests/testdata/cpp-multimodule-project/sonar-project.properties
> <https://github.com/wenns/sonar-cxx/pull/358#discussion_r28641725>:
>
> > @@ -0,0 +1,56 @@
> > +sonar.host.url=http://localhost:9000
> > +
> > +sonar.modules=cli,lib,package1,package2
> > +cli.sonar.projectBaseDir=cli
>
> Are all these settings (cli, doxygen, ...) needed or can they removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wenns/sonar-cxx/pull/358/files#r28641725>.
>